h/static/scripts/annotator/plugin/test/pdf-metadata-test.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+var PDFMetadataService = require('../pdf-metadata');
+
+describe('pdf-metadata', function () {
+  var pdfMetadata;
+  var fakePDFViewerApplication = {
+    'documentFingerprint': 'fakeFingerprint',
+    'documentInfo': {
+      'Title': 'fakeTitle',
+    },
+    'metadata': {
+      'metadata': {
+        'dc:title': 'fakeTitle',
+      }
+    },
+    'url': 'fakeUrl',
+  };
+  var sandbox;
+
+  beforeEach(function () {
+    pdfMetadata = new PDFMetadataService(fakePDFViewerApplication);
+    pdfMetadata._documentLoaded = Promise.resolve();
```

Comment:
Setting `_documentLoaded` directly means that we're not exercising the code that listens for the relevant event. 

You can simulate events with:

```
var event = document.createEvent('Event')
event.initEvent('documentload', false /* bubbles */, false /* cancelable */);
window.dispatchEvent(event);
```
(In an actual browser the first two lines can be replaced with just `var event = new Event(<event type>)` but that is not supported in PhantomJS)